semanticdb-gradle-plugin/src/main/scala/SemanticdbGradlePlugin.scala

```diff
@@ -396,8 +400,43 @@ class WriteDependencies extends DefaultTask {
       .foreach(path => java.nio.file.Files.createDirectories(path.getParent()))
 
     val deps = List.newBuilder[String]
+    val project = getProject()
+
+    // List the project itself as a dependency so that we can assign project name/version to symbols that are defined in this project.
+    // The code below is roughly equivalent to the following with Groovy:
+    //   deps += "$publication.groupId $publication.artifactId $publication.version $sourceSets.main.output.classesDirectory"
+    Try(
```

Comment:
I think we should log out the exception, but not let it propagate. Even though this feature is optional, having some feedback from failures would be good especially when investigating code we don't have access to.